getting-started/create-a-mobile-property.md

```diff
@@ -48,19 +48,17 @@ Not sure on what extensions you need? Check out the extensions in the _Using Mob
 
 ## Publish the configuration
 
-To create a library of changes and deploy the library to a **Development Environment**:
+To create a library of changes and publish them to a **Development Environment**, complete the following steps:
 
-1. In the **Publishing** tab, under the **Development** section of the publishing workflow.
-
-   click **Add New Library**.
-
-2. Specify a name for the library and, from the **Environment** drop-down list, select a development environment .
-3. Add the configuration changes to be deployed.
-4. Click **Add All Changed Resources.**  To add only some changes, click **Add a Resource**. 
+1. Click the **Publishing** tab.
+2. Under the **Development** section of the publishing workflow, click **Add New Library**.
+3. Specify any name for the library **Name**.
+4. From the **Environment** drop-down list, select Development as the environment.
+5. Click **Add All Changed Resources** to add the configuration changes to be deployed. (To add only some changes, click **Add a Resource**.) 
 5. Click **Save & Build for Development**.   **Tip**: The library builds and is displayed under the **Development** section of the publishing workflow.
-6. Click on the down arrow for the library and select **Submit for Approval**.
+6. On the library card, click *...* to see a dropdown and select **Submit for Approval** and then **Submit**.
```

Comment:
As this step involves a few substeps, the content needs to be split into several steps:
"6. On the library card, click *...* to see a drop-down list.
7. Select **Submit for Approval** and then **Submit**."